src/vs/workbench/api/browser/mainThreadDebugService.ts

```diff
@@ -225,6 +228,13 @@ export class MainThreadDebugService implements MainThreadDebugServiceShape, IDeb
 		});
 	}
 
+	public $setDebugSessionName(sessionId: DebugSessionUUID, name: string): void {
+		const session = this.debugService.getModel().getSession(sessionId, true);
```

Comment:
No need to pass `true` here.
Inactive sessions are dead sessions which are still only shown in the Debug Console. I think we should not support renaming them.